Traceback (most recent call last):
  File "/w/284/murdock/merge/benchmark.py", line 4, in <module>
    from datasets import load_dataset
ModuleNotFoundError: No module named 'datasets'
srun: error: gpunode7: task 0: Exited with exit code 1
